entity-framework/core/miscellaneous/async.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Asynchronous Programming - EF Core
+description: Querying and saving data asynchronously with Entity Framework Core
+author: roji
+ms.date: 9/2/2020
+ms.assetid: 38f71624-7a68-4c72-a918-3e7b858ef090
+uid: core/miscellaneous/async
+---
+# Asynchronous Programming
+
+Asynchronous queries avoid blocking a thread while the query is executed in the database. Async queries are important for keeping a responsive UI in thick-client applications, and can also increase throughput in web applications where they free up the thread to service other requests in web applications.
```

Comment:
If merged this should not refer to query.

Reply:
I don't think "thick-client" is a thing, change to "rich client"

Reply:
Ah, the nineties were a good time.